### ISSUE TITLE
Installing Tackle on RPI4

### ISSUE THREAD

**Issue by chengkuangan** [konveyor/operator#138]
Hi SME, 

I tried to install Tackle on RPI4 but it shows the following error in the subscription:
```
bundle unpacking failed. Reason: BackoffLimitExceeded, and Message: Job has reached the specified backoff limit
```
Tried the [steps](https://access.redhat.com/solutions/6459071) here, problem remains.

I am using OLM v0.22.0 and Kubernetes v1.24.2

Can Tackle be installed on RPI4?

**Comment by fbladilo** [comment]
@chengkuangan Your k8s version and OLM seem reasonable but without more information about other resources in the namespace is hard to say what is happening , specifically state of operator/operands, also full subscription output.

**Comment by jmontleon** [comment]
@chengkuangan This may be possible now, but it's untested. We've done multi-arch builds for amd64 and aarch64 for all of the images. Prior to this work (started in February) there would have been no images available that would run properly on an RPI4 cluster. 

The bundle image isn't architecture specific, however so I'm not sure what would be happening at that point to cause a problem. If you try again and hit the same issue it would be interesting to know if the catalog source resource has any status indicating a problem, or even the pod to extract the bundle.

**Comment by chengkuangan** [comment]
@jmontleon I suspected it was no `arm64` earlier when I was trying.  I will look into trying this again and will get back. Thank you for notifying me.

**Comment by jmontleon** [comment]
@chengkuangan Thanks. The operator is called `Konveyor Operator` now and we released 0.1.0 of Konveyor Operator yesterday. If you give it a try and have problems let us know and if we can find a way around it we can fix it.

**Comment by djzager** [comment]
Since we don't have a bot to mark issues as stale and isn't something currently on our roadmap but something that may work. I'm going to close the issue, trusting it will be reopened if it is an actual problem we should look into solving.
